zbus_macros/src/lib.rs

```diff
@@ -363,6 +364,68 @@ pub fn proxy(attr: TokenStream, item: TokenStream) -> TokenStream {
 /// # Ok::<_, Box<dyn Error + Send + Sync>>(())
 /// ```
 ///
+/// # Synchronous between multiple interfaces
+///
+/// If a server host multiple interfaces, it is possible to synchronous between them.
+///
+/// ```no_run
+/// use zbus_macros::interface;
+/// # const PATH: &str = "/org/freedesktop/zbus_macros/test_iface_sync";
+/// # const WELL_KNOWN_NAME: &str = "org.freedesktop.zbus_macros_iface_sync";
+/// struct ClientA;
+/// struct ClientB {
+///     foo: u32,
+/// }
+///
+/// #[interface(proxy, name = "org.freedesktop.zbus.ClientA")]
```

Comment:
nitpick: interfaces don't represent clients so I think this is not a good name. How about `InterfaceA` instead? Same for `ClientB`.

Reply:
Renamed `ClientA` -> `InterfaceA`, `ClientB` -> `InterfaceB`.